subprojects/core-api/src/main/java/org/gradle/api/provider/ProviderFactory.java

```diff
@@ -254,7 +254,6 @@ public interface ProviderFactory {
      * @return the provider, never returns null
      * @since 6.1
      */
-    @Incubating
```

Comment:
Let's not promote the `ValueSource` stuff yet. We'll do this in 8.1 instead once we've finalized the contract.